<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>604</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hand Reader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PDGUI.qrc">
    <normaloff>:/icons/resources/icons/vsign.png</normaloff>:/icons/resources/icons/vsign.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="IplImageWidget" name="widget01" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="IplImageWidget" name="widget10" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="IplImageWidget" name="widget11" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="IplImageWidget" name="widget00" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionToggleWebcam"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Learning</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLearnPostures"/>
    <addaction name="actionLearnBackground"/>
    <addaction name="actionLearnSkin"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionFullScreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Settings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleWebcam"/>
   <addaction name="actionFullScreen"/>
   <addaction name="separator"/>
   <addaction name="actionLearnPostures"/>
   <addaction name="actionLearnBackground"/>
   <addaction name="actionLearnSkin"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionToggleWebcam">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PDGUI.qrc">
     <normaloff>:/icons/resources/icons/webcam.png</normaloff>:/icons/resources/icons/webcam.png</iconset>
   </property>
   <property name="text">
    <string>Start/Stop &amp;Webcam</string>
   </property>
  </action>
  <action name="actionOpenVideo">
   <property name="icon">
    <iconset resource="PDGUI.qrc">
     <normaloff>:/icons/resources/icons/film.png</normaloff>:/icons/resources/icons/film.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;Video</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="PDGUI.qrc">
     <normaloff>:/icons/resources/icons/exit.png</normaloff>:/icons/resources/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="PDGUI.qrc">
     <normaloff>:/icons/resources/icons/info.png</normaloff>:/icons/resources/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PDGUI.qrc">
     <normaloff>:/icons/resources/icons/fullscreen.png</normaloff>:/icons/resources/icons/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Full Screen</string>
   </property>
  </action>
  <action name="actionLearnBackground">
   <property name="icon">
    <iconset resource="PDGUI.qrc">
     <normaloff>:/icons/resources/icons/background.png</normaloff>:/icons/resources/icons/background.png</iconset>
   </property>
   <property name="text">
    <string>Learn &amp;Background</string>
   </property>
  </action>
  <action name="actionLearnSkin">
   <property name="icon">
    <iconset resource="PDGUI.qrc">
     <normaloff>:/icons/resources/icons/hand.png</normaloff>:/icons/resources/icons/hand.png</iconset>
   </property>
   <property name="text">
    <string>Learn S&amp;kin</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="PDGUI.qrc">
     <normaloff>:/icons/resources/icons/options.png</normaloff>:/icons/resources/icons/options.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="actionLearnPostures">
   <property name="icon">
    <iconset resource="PDGUI.qrc">
     <normaloff>:/icons/resources/icons/learn.png</normaloff>:/icons/resources/icons/learn.png</iconset>
   </property>
   <property name="text">
    <string>Learn &amp;Postures</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IplImageWidget</class>
   <extends>QWidget</extends>
   <header>iplimagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PDGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFullScreen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleWebcam</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleWebcam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOptions</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openOptions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLearnBackground</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>openLearnBg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLearnSkin</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>openLearnSkin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLearnPostures</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openLearnPostures()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleFullScreen()</slot>
  <slot>openAbout()</slot>
  <slot>toggleWebcam()</slot>
  <slot>openOptions()</slot>
  <slot>openLearnBg()</slot>
  <slot>openLearnSkin()</slot>
  <slot>openLearnPostures()</slot>
 </slots>
</ui>
